src/ert/job_queue/queue.py

```diff
@@ -11,16 +11,7 @@
 import time
 from collections import deque
 from threading import BoundedSemaphore, Semaphore
-from typing import (
-    TYPE_CHECKING,
-    Callable,
-    Dict,
-    List,
-    Optional,
-    Sequence,
-    Tuple,
-    Union,
-)
+from typing import TYPE_CHECKING, Callable, Dict, List, Optional, Sequence, Tuple, Union
```

Comment:
which tool conflicts with which when we get this back-and-forth shuffling of the formatting?

Reply:
This is the latest formatting confirmed by black and ruff.

Reply:
Only if the trailing comma in the multiline version is removed first. If that comma is kept, both black and ruff at their most recent versions will not touch the formatting.

Reply:
No, regardless the comma. This is what will yield.